src/binfmt/binfmt.c

```diff
@@ -27,76 +27,81 @@ static bool arg_cat_config = false;
 static PagerFlags arg_pager_flags = 0;
 static bool arg_unregister = false;
 
-static int delete_rule(const char *rule) {
-        _cleanup_free_ char *x = NULL, *fn = NULL;
-        char *e;
+static int delete_rule(const char *rulename) {
+        const char *fn = strjoina("/proc/sys/fs/binfmt_misc/", rulename);
+        return write_string_file(fn, "-1", WRITE_STRING_FILE_DISABLE_BUFFER);
+}
 
+static int apply_rule(const char *filename, unsigned line, const char *rule) {
+        assert(filename);
+        assert(line > 0);
         assert(rule);
         assert(rule[0]);
 
+        _cleanup_free_ char *rulename = NULL;
+        const char *e;
+        int r;
+
         e = strchrnul(rule + 1, rule[0]);
-        x = strndup(rule + 1, e - rule - 1);
-        if (!x)
+        rulename = strndup(rule + 1, e - rule - 1);
+        if (!rulename)
                 return log_oom();
 
-        if (!filename_is_valid(x) ||
-            STR_IN_SET(x, "register", "status"))
+        if (!filename_is_valid(rulename) ||
+            STR_IN_SET(rulename, "register", "status"))
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
-                                       "Rule file name '%s' is not valid, refusing.", x);
-
-        fn = path_join("/proc/sys/fs/binfmt_misc", x);
-        if (!fn)
-                return log_oom();
-
-        return write_string_file(fn, "-1", WRITE_STRING_FILE_DISABLE_BUFFER);
-}
-
-static int apply_rule(const char *rule) {
-        int r;
-
-        (void) delete_rule(rule);
+                                       "%s:%u: rule name '%s' is not valid, refusing.",
```

Comment:
we usually start our sentences with an uppercase letter? And we used to here, too? why change?

Reply:
Because the sentence is prefixed by the file name. But I guess it's indeed better to keep it uppercased, I'll revert that bit.